### PR TITLE
feat: Import Screen

### PR DESCRIPTION
**Pull request by david-contractor**
- Created Upload screen based on Figma
- Route
- Menu Item
- Screen
- Added react-dropzone package

### CODE REVIEW

File: src/components/import/DropZone.tsx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+  const uploadFileMutation = useMutation({
+    mutationFn: uploadFile,
+    onSuccess: (data, fileObj) => {
+      setUploadStatus(fileObj, UploadStatus.Complete)
+    },
+    onError: (error: AxiosError, data) => {
+      console.log(`Error uploading files: ${error.message}`)
+      setUploadStatus(data, UploadStatus.Failed)
+    }
```
Comment (marrouchi) [inline src/components/import/DropZone.tsx:108]: We could use the snackbar in order to show upload success or failure message to the user.

File: src/components/import/Import.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <Container>
+      <Breadcrumbs>
+        <Link underline="hover" color="inherit" href="/">
+          Home
+        </Link>
+        <Typography color="text.primary">Import</Typography>
+      </Breadcrumbs>
+      <Typography color="text.primary" variant="h4">
+        Import
+      </Typography>
```
Comment (marrouchi) [inline src/components/import/Import.tsx:22]: You could re-use the PageHeader component to display the title and breadscrumbs.

File: src/components/import/UploadFileListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          valueBuffer={100}
+        />
+      </Grid>
+      <Grid
+        item
+        xs={1}
+        justifyContent="center"
+        alignItems="center"
+        textAlign="center"
+      ></Grid>
```
Comment (marrouchi) [inline src/components/import/UploadFileListItem.tsx:93]: I wonder if we need those empty grid components ?

File: src/components/import/UploadFileListItem.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+const UploadFileListItem: FC<FileObjProps> = (props: FileObjProps) => {
+  return (
+    <Grid
+      container
+      justifyContent="center"
+      alignItems="center"
+      spacing={0}
+      className={`import__upload-list-item ${
+        props.fileObj.status === UploadStatus.Failed ? 'failed' : ''
+      }`}
```
Comment (marrouchi) [inline src/components/import/UploadFileListItem.tsx:25]: It seems there's a code repetition here, maybe declare a var for it  (DRY) ?

File: src/components/import/Import.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  border: 1px dashed rgba(0, 0, 0, 0.12);
+  border-radius: 4px;
+  max-width: 600px;
+  max-height: 152px;
+  text-align: center;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+.dropzone-div {
```
Comment (marrouchi) [inline src/components/import/Import.css:14]:
```suggestion
.dropzone-inner {
```

File: src/components/import/DropZone.tsx [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+  const handleCancel = (): void => {
+    setFilesObj([])
+  }
+
+  const handleUpload = () => {
+    fileObjs.forEach(async fileObj => {
+      if (
+        fileObj.status === UploadStatus.Complete ||
+        fileObj.status === UploadStatus.Failed
```
Comment (marrouchi) [inline src/components/import/DropZone.tsx:119]: Does it include the files currently being uploaded (concurrency) ?

**Review by marrouchi — CHANGES_REQUESTED** [review]
Looks good, left a couple of comments.

**Review by marrouchi — APPROVED** [review]

File: src/components/import/DropZone.tsx (hunk 6 above)
Comment (david-contractor) [reply src/components/import/DropZone.tsx:156]: Disabling attach during an upload operation as discussed.

**Pull request merged**